package.json

```diff
@@ -33,6 +32,7 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.11",
```

Comment:
this is cool, I think we should add it by default to the challenge, what do you think @scottlepich ?